static/js/benchmark.js

```diff
@@ -95,6 +95,8 @@
 
   function draw(drawDiv) {
     return function(datatable) {
+
+      var valueRange = $('#' + datatable.header).data('value-range') || [0,null];
```

Comment:
don't create a new selector based on id, you have the DOM element already as `drawDiv` (which is a strange variable name anyway)